x-pack/legacy/plugins/reporting/public/components/report_info_button.tsx

```diff
@@ -75,52 +75,51 @@ export class ReportInfoButton extends Component<Props, State> {
       return null;
     }
 
-    const jobType = get(info, 'jobtype', NA);
-    const processedBy =
-      has(info, 'kibana_name') && has(info, 'kibana_id')
-        ? `${info.kibana_name} (${info.kibana_id})`
-        : UNKNOWN;
+    const jobType = info.jobtype || NA;
 
     // TODO queue method (clicked UI, watcher, etc)
     const jobInfoParts = {
       datetimes: [
         {
           title: 'Created By',
-          description: get(info, 'created_by', NA),
+          description: info.created_by || NA,
         },
         {
           title: 'Created At',
-          description: get(info, 'created_at', NA),
+          description: info.created_at || NA,
         },
         {
           title: 'Started At',
-          description: get(info, 'started_at', NA),
+          description: info.started_at || NA,
         },
         {
           title: 'Completed At',
-          description: get(info, 'completed_at', NA),
+          description: info.completed_at || NA,
         },
         {
           title: 'Processed By',
-          description: processedBy,
+          description:
+            info.kibana_name && info.kibana_id
+              ? `${info.kibana_name} (${info.kibana_id})`
+              : UNKNOWN,
         },
         {
           title: 'Browser Timezone',
-          description: get(info, 'payload.browserTimezone', NA),
+          description: (info.payload && info.payload.browserTimezone) || NA,
```

Comment:
@joelgriffith with the new version of TS `get()` returns `unknown` type values, which is problematic for logic consuming these values, so I updated things to just use the property directly which doesn't look too bad if you ask me. One thing I wasn't sure about is if the `info.payload` property is actually guaranteed to be there like is implied in the typedef for `JobInfo` in `x-pack/legacy/plugins/reporting/public/lib/job_queue_client.ts`, so I make `payload`, `output`, and `meta` optional there. Are they for sure going to be there? Can I make these property accesses even simpler?

Reply:
@spalger yes -- I see elsewhere in the codebase where we directly reference the `payload `property without doing a conditional check. I think your `description` adjustment is good, and I'm always a +1 for getting more types out.